Sources/dojo-ios-sdk-drop-in-ui/Classes/Entities/DojoThemeSettings.swift

```diff
@@ -73,6 +77,8 @@ public class DojoThemeSettings: NSObject {
         self.inputFieldDefaultBorderColor = inputFieldDefaultBorderColor ?? UIColor.init(hexaARGB: "#26000000") ?? .gray
         self.errorTextColor = errorTextColor ?? UIColor.init(hexaARGB: "#FFB00020") ?? .systemRed
         self.showBranding = true
+        self.backdropViewColor = backdropViewColor ?? UIColor.black.withAlphaComponent(0.6)
```

Comment:
isn't the alpha already controlled by `backdropViewAlpha` ?

Reply:
good one, fixed